src/grip/story.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
```

Comment:
With MDX you can merge story and md files

Reply:
Thanks!